### PR TITLE
Refactor ids model enhancement (directly reuse external concepts instead of copying)

### PR DESCRIPTION
**Pull request by JohannesLipp**
Please see https://docs.google.com/spreadsheets/d/17RjLl6wkL_V2_jbf_vBfJ_77NgmENnf_tIfWZsNhr_Y/edit#gid=0 for details.

### CODE REVIEW

**Comment by HaydarAk** [comment]
Looks good for me, at least from an RDF perspective :)  

Please resolve the two questions / fixes.

**Review by HaydarAk — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,12 @@ artifacts
...
 *.project
+model/content/.DS_Store
+model/governance/.DS_Store
+model/.DS_Store
+model/.DS_Store
+.DS_Store
+model/.DS_Store
+model/.DS_Store
+.DS_Store
+model/.DS_Store
```
[the remark this answers is not in the document]
Comment (JohannesLipp) [reply .gitignore]: Thanks! Removed the duplicates and adopted to `.DS_Store` based on the template from https://www.toptal.com/developers/gitignore/api/macos

File: utils/refactor_helper.ttl [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+
+time:TemporalEntity
+    a owl:Class ;
+    rdfs:label "Temporal entity"@en ;
+    rdfs:comment "A temporal interval, instant or a relative duration."@en .
+
+# Definition of Properties
+
+dct:title
+    a rdf:Property ;
```
[the remark this answers is not in the document]
Comment (pnlinden) [reply utils/refactor_helper.ttl:169]: I would need to add one or two lines of code to our context generation, but other than that the CodeGen Tool can handle this :)

**Comment by HaydarAk** [comment]
Important note (and the reason why I revoked my approval) 
We have re-evaluate the policy-related classes and especially the codes.

The `ids:Action`, `ids:BinaryOperator` and maybe other odrl-based classes have some instances which are equivalent to the odrl counterpart while others are more precise or new. Until now this was not a concern due to the fact taht we use the `idsc` namespace and `subClassOf` relation. 
The reevaluation may result in two sets of actions (for example). 
some ids:Actions and some odrl:Action.

**Comment by JohannesLipp** [comment]
@HaydarAk @pnlinden Please review the current state as I incorporated Haydar's request for changes w.r.t. the policies and codes. Thank you!

**Pull request merged**